core-lib/TestSuite/ActorTests.ns

```diff
@@ -206,6 +214,22 @@ class ActorTests usingPlatform: platform testFramework: minitest = Value (
       ^ pp promise
     )
 
+    public testAsyncAssertionFailed= (
```

Comment:
Wouldn't this test fit much better into the [MinitestTests](https://github.com/smarr/SOMns/blob/dev/core-lib/TestSuite/MinitestTests.ns#L215)?
It seems misplaced here.